forge-gui/res/cardsfolder/upcoming/cone_of_cold.txt

```diff
@@ -0,0 +1,13 @@
+﻿Name:Cone of Cold
+ManaCost:3 U
+Types:Sorcery
+A:SP$ RollDice | Sides$ 20 | ResultSubAbilities$ 1-9:TapAll,10-19:TapAndLock,20:TapLock2 | SpellDescription$ Roll a d20.
+SVar:TapAll:DB$ TapAll | ValidCards$ Creature.OppCtrl | StackDescription$ SpellDescription | SpellDescription$ 1—9 VERT Tap all creatures your opponents control.
+SVar:TapAndLock:DB$ TapAll | ValidCards$ Creature.OppCtrl | SubAbility$ DBNoUntap | StackDescription$ SpellDescription | SpellDescription$ 10—19 VERT Tap all creatures your opponents control. Those creatures don't untap during their controllers' next untap steps.
+SVar:DBNoUntap:DB$ PumpAll | ValidCards$ Creature.OppCtrl | KW$ HIDDEN This card doesn't untap during your next untap step. | IsCurse$ True | Duration$ Permanent
+SVar:TapLock2:DB$ TapAll | ValidCards$ Creature.OppCtrl | SubAbility$ NoUntap2 | StackDescription$ SpellDescription | SpellDescription$ 20 VERT Tap all creatures your opponents control. Those creatures don't untap during their controllers' next untap steps.
+SVar:NoUntap2:DB$ PumpAll | ValidCards$ Creature.OppCtrl | KW$ HIDDEN This card doesn't untap during your next untap step. | IsCurse$ True | Duration$ Permanent | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | TgtZone$ Battlefield | ValidCards$ Creature.OppCtrl | ReplacementEffects$ TapLockETBTapped | Name$ Cone of Cold Effect | Duration$ UntilYourNextTurn | StackDescription$ SpellDescription | SpellDescription$ 20 VERT Tap all creatures your opponents control. Those creatures don't untap during their controllers' next untap steps. Until your next turn, creatures your opponents control enter the battlefield tapped.
+SVar:TapLockETBTapped:Event$ Moved | ValidCard$ Creature.OppCtrl | Destination$ Battlefield | ReplaceWith$ ETBTapped | ActiveZones$ Battlefield | SubAbility$ TapAndLock
+SVar:ETBTapped:DB$ ChangeZone | Origin$ All | Destination$ Battlefield | Tapped$ True | Defined$ ReplacedCard
```

Comment:
check #722 for the new ReplaceTapped

Also the ReplacementEffect shouldn't have `| ActiveZones$ Battlefield` , that is set intern by `Effect` above